### PR TITLE
LPD-26339 - Add default language labels missing during import

### PR DESCRIPTION
**Pull request by evibudai**
[LPD-26339](https://liferay.atlassian.net/browse/LPD-26339)
Hi Guys,

I did the changes @marcelabc requested on this PR: https://github.com/liferay-objects/liferay-portal/pull/3367

1. I removed the _addDefaultLocaleLabel method and used computeIfAbsent()

CC @G-Sa

### CODE REVIEW

File: modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java [hunk 1]
```diff
@@ -908,6 +918,16 @@ private void _addObjectDefinitionResources(
 								objectRelationship.getName());
 				}
 
+				Map<String, String> localeLabelMap =
+					objectRelationship.getLabel();
+
+				String objectRelationshipName = objectRelationship.getName();
+
+				localeLabelMap.computeIfAbsent(
+					defaultLocale, key -> objectRelationshipName);
```
Comment (G-Sa) [inline modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java:927]: You can inline the `objectRelationshipName` just as you did with the `objectAction.getName()` @evibudai

Comment (G-Sa) [reply modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java:927]: Nevermind, @marcelabc pointed out that this variable needs to be final/efectively final. So ignore my comment xD

Comment (marcelabc) [reply modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java:927]: She can't do it here. It needs to be a final variable if I'm not mistaken.

File: modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java [hunk 2]
```diff
@@ -908,6 +918,16 @@ private void _addObjectDefinitionResources(
 								objectRelationship.getName());
 				}
 
+				Map<String, String> localeLabelMap =
+					objectRelationship.getLabel();
```
Comment (marcelabc) [inline modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java:922]: Please rename it to labelMap.

**Review by marcelabc — CHANGES_REQUESTED** [review]
LGTM 🎉

File: modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java [hunk 3]
```diff
@@ -842,6 +844,7 @@ private void _addListTypeDefinition(ObjectDefinition objectDefinition)
 	}
 
 	private void _addObjectDefinitionResources(
+			String defaultLocale,
```
Comment (marcelabc) [inline modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java:847]: Please rename it to `defaultLanguageId` and reorder the parameters.

File: modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java [hunk 4]
```diff
@@ -865,6 +868,13 @@ private void _addObjectDefinitionResources(
 							objectAction.getExternalReferenceCode(),
 							objectDefinitionId);
 
+				Map<String, String> localeLabelMap = objectAction.getLabel();
+
+				localeLabelMap.computeIfAbsent(
+					defaultLocale, key -> objectAction.getName());
+
+				objectAction.setLabel(() -> localeLabelMap);
```
Comment (marcelabc) [inline modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java:876]: Can you replace this code to:
```
Map<String, String> labelMap = objectRelationship.getLabel();
    
ObjectRelationship finalObjectRelationship = objectRelationship;

labelMap.computeIfAbsent(
	defaultLanguageId, key -> finalObjectRelationship.getName());

objectRelationship.setLabel(() -> labelMap);
```

File: modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java [hunk 5]
```diff
@@ -908,6 +918,16 @@ private void _addObjectDefinitionResources(
...
+				Map<String, String> localeLabelMap =
+					objectRelationship.getLabel();
+
+				String objectRelationshipName = objectRelationship.getName();
+
+				localeLabelMap.computeIfAbsent(
+					defaultLocale, key -> objectRelationshipName);
+
+				objectRelationship.setLabel(() -> localeLabelMap);
+
```
Comment (marcelabc) [inline modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java:930]: Can you replace this code to:
```
Map<String, String> labelMap = objectRelationship.getLabel();
    
ObjectRelationship finalObjectRelationship = objectRelationship;

labelMap.computeIfAbsent(
	defaultLanguageId, key -> finalObjectRelationship.getName());

objectRelationship.setLabel(() -> labelMap);
```

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
Analyzing CI

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:stop

**Comment by marcelabc** [comment]
Hey @evibudai 
The test is failing. I resent it here https://github.com/liferay-objects/liferay-portal/pull/3377.

**Pull request closed without merging**